product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx

```diff
@@ -5,56 +5,71 @@ navTitle: "Prerequisites"
 
 Postres Enterprise Manager requires a Postgres server running either [EDB Postgres Advanced Server](/epas/latest) or [PostgresSQL](/supported-open-source/postgresql/) version 11 or later to hold the Postgres Enterprise Manager backend database. 
   
-  1. Install the Postgres server. See [Installing EDB Postgres Advanced Server on Linux](/epas/latest/epas_inst_linux) or [Installing PostgreSQL](/supported-open-source/postgresql/installer/).
+  1. Set up the repository
+
+     Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+
+     -  To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+
+     -  To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+
+
+  2. Install the Postgres server. See [Installing EDB Postgres Advanced Server on Linux](/epas/latest/epas_inst_linux) or [Installing PostgreSQL](/supported-open-source/postgresql/installer/).
   
-  2. Configure authentication on the Postgres server by updating the `pg_hba.conf` file. The `pg_hba.conf` file manages connections. Verify that the `pg_hba.conf` file allows connections from the PEM server, the monitoring PEM agent, and the host of the Apache web server server.
+  3. Configure authentication on the Postgres server by updating the `pg_hba.conf` file. The `pg_hba.conf` file manages connections. Verify that the `pg_hba.conf` file allows connections from the PEM server, the monitoring PEM agent, and the host of the Apache web server server. Pointers for configuring access:
   
+     - PEM server connects to the PEM backend database as a superuser using password authentication. On Linux platforms, this requires you to add a new superuser that authenticates using a password.
+     
+     - To allow the new superuser to connect using password authentication, add a line `pg_hba.conf` that allows `host` connections using `md5` authentication.
```

Comment:
I should have said md5 *or scram-sha-256* authentication. The default changed in pg14 I think.

Reply:
updated and committed